scripts/intfutil

```diff
@@ -23,42 +25,75 @@ PORT_DESCRIPTION = "description"
 PORT_OPTICS_TYPE = "type"
 PORT_PFC_ASYM_STATUS = "pfc_asym"
 
+def db_connect_configdb():
+    """
+    Connect to configdb
+    """
+    config_db = ConfigDBConnector()
+    if config_db is None: 
+        return None 
+    config_db.connect()
+    return config_db 
+
+
+def get_interface_vlan_dict(config_db):
+    """
+    Get info from REDIS ConfigDB and create interface to vlan mapping
+    """
+    get_int_vlan_configdb_info = config_db.get_table('VLAN_MEMBER')
+    int_list = []
+    vlan_list = []
+    for line in get_int_vlan_configdb_info:
+        vlan_number = line[0]
+        interface = line[1]
+        int_list.append(interface)
+        vlan_list.append(vlan_number)
+    int_to_vlan_dict = dict(zip(int_list, vlan_list))
+    return int_to_vlan_dict
+
+
+def config_db_vlan_port_keys_get(int_to_vlan_dict, intf_name):
+    """
+    Get interface vlan value and return it.
+    """
+    vlan = "routed"
+    if intf_name.startswith("Ethernet"):
+        if intf_name in int_to_vlan_dict.keys():
+            vlan = int_to_vlan_dict[intf_name] 
+    return vlan
+
 
 def db_connect_appl():
     appl_db = swsssdk.SonicV2Connector(host='127.0.0.1')
     if appl_db is None:
         return None
-
     appl_db.connect(appl_db.APPL_DB)
-
     return appl_db
 
 
 def appl_db_keys_get(appl_db, intf_name):
-
+    """
+    Get APPL_DB Keys
+    """
     if intf_name is None:
         appl_db_keys = appl_db.keys(appl_db.APPL_DB, "PORT_TABLE:*")
     elif intf_name.startswith('Ethernet'):
```

Comment:
Please also fix this existing code as per above request.

Reply:
fixed